ts/components/wallet/paymentSummary/UpdatedPaymentSummaryComponent.tsx

```diff
@@ -0,0 +1,119 @@
+/**
+ * This component will display the transaction details if updates on the amount are identified
+ */
+import { H1, H3, Icon, Right, Text, View } from "native-base";
+import * as React from "react";
+import { StyleSheet } from "react-native";
+import { Grid, Row } from "react-native-easy-grid";
+import { NavigationScreenProp, NavigationState } from "react-navigation";
+import I18n from "../../../i18n";
+import variables from "../../../theme/variables";
+
+type Props = Readonly<{
+  navigation: NavigationScreenProp<NavigationState>;
+  amount: string;
+  updatedAmount: string;
+  expireDate: string;
+  tranche: string;
+}>;
+
+const styles = StyleSheet.create({
+  padded: {
+    paddingRight: variables.contentPadding,
+    paddingLeft: variables.contentPadding
+  },
+
+  darkContent: {
+    backgroundColor: variables.brandDarkGray
+  },
+
+  white: {
+    color: variables.brandLight
+  },
+
+  whiteStrike: {
```

Comment:
some of these styles appear to be recurring in multiple places throughout the code: consider moving them to `ts/components/styles/wallet.ts`